Sources/networkcommunication/ALAppLocalNotifications.m

```diff
@@ -106,11 +106,17 @@ - (void)reachabilityChanged:(NSNotification*)note {
 
 - (void)proactivelyConnectMQTT {
     ALMQTTConversationService *alMqttConversationService = [ALMQTTConversationService sharedInstance];
+    NSString *title = @"support-channel-";
```

Comment:
If we are only subscribing to this topic from agent app, then should we subscribe in this method? It will be executed for SDK users as well. Wouldn't it be better to create an interface from here and use it from agent app.

Reply:
@mukeshthawani  its needed for SDK as well.

Reply:
For SDK also, realtime update of agent status is not working for end user. same topic will be responsible for agent status updation.

Reply:
In that case, we should subscribe to this topic from SDK's Conversation VC as well. Currently, we are only doing it in the agent app.

Reply:
Okay. Hard coding to "support-channel-" in this core file is not a good idea. See if there's a way to pass the topic from outside(SDK and app).